StudyPathTraker/StudyPathTraker/Navigation/SPLightNavigationController.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  SPLightNavigationController.swift
+//  StudyPathTraker
+//
+//  Created by Rafael Lopez on 5/30/18.
+//  Copyright © 2018 Jerti. All rights reserved.
+//
+
+import UIKit
+
+class SPLightNavigationController: UINavigationController {
```

Comment:
why are you subclassing navigation controller?

Reply:
Moved as an extension